src/Microsoft.Health.Dicom.Core/Serialization/Newtonsoft/ConfigurationJsonConverter.cs

```diff
@@ -0,0 +1,111 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Linq;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
```

Comment:
Can't we use System.Text.Json?

Reply:
The Durable Task framework takes a dependency on Newtonsoft.Json for its serialization, but one day I'll go make that change too 😛